dashboard/views.py

```diff
@@ -150,3 +152,36 @@ def post(self, request):
         return Response(
             data=enrollment.json
         )
+
+class UserPaymentStatus(APIView):
+    """
+    returns list of program id and status weather user has paid for atleast one course in selected program.
```

Comment:
Please change to `API view for determining whether a user has paid for any course run in their enrolled programs`